README.md

```diff
@@ -1 +1,42 @@
-# veres-one-context
\ No newline at end of file
+# Veres One Context _(veres-one-context)_
+
+> JSON-LD Context for Veres One ledger
+
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Commercial Support](#commercial-support)
+- [License](#license)
+
+## Install
+
+Requires Node.js.
+
+To install via NPM:
+
+```
+npm install veres-one-context
+```
+
+## Usage
+
+```
+const contextV1 = require('veres-one-context')
+
+contextV1['@context']['@version']  // -> "1.1"
```

Comment:
What's this about? This seems confusing ... this `@version` is referring to JSON-LD and not veres one, but this text makes it seem like it's telling the reader it's referring to veres one.

Reply:
Hehe, sorry, was just trying to demonstrate that it's parsed JSON, so I picked the very first properties in the object. I'll use something else. :)